packages/quick_actions/quick_actions_android/test/quick_actions_android_test.dart

```diff
@@ -0,0 +1,155 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:quick_actions_android/quick_actions_android.dart';
+import 'package:quick_actions_platform_interface/quick_actions_platform_interface.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('$QuickActionsAndroid', () {
+    final QuickActionsAndroid quickActions = QuickActionsAndroid();
```

Comment:
I missed this on the iOS side, but probably applies there too: please put this line in each test. Having the object under test be stateful across the tests is an antipattern and leads to a variety of problems (including tests passing when they shouldn't).